specs/mediawiki/v1/content.yaml

```diff
@@ -75,7 +70,7 @@ paths:
         # See also http://tools.ietf.org/html/rfc7231#section-7.1.2
         uri: /{domain}/sys/parsoid/html/{title}/latest
 
-  /{module:page}/{title}/html/:
+  /{module:page}/html/{title}/:
```

Comment:
The more I think about this from the content model perspective, the more I'm leaning towards listing the *potential* revisions that can have an entry in a given property. For HTML, that would probably be all of them. For data-parsoid, that would only be the revisions with a parsoid-compatible content model.